lib/membrane_h264_plugin/parser/au_splitter.ex

```diff
@@ -143,9 +143,9 @@ defmodule Membrane.H264.Parser.AUSplitter do
   These NAL units aren't proved to form a new access units and that is why they haven't yet been
   output by `split_nalus`.
```

Comment:
Please check whether `split_nalus` is linked correctly after doing `mix docs`

I think that we could also mention that the splitter state is reset

Reply:
done, however the docs configuration is yet to be revisited